### PR TITLE
Unify code for sending signals to searchmanager

### PR DESCRIPTION
**Pull request by slomo**

### CODE REVIEW

**Comment by slomo** [comment]
@FabiApfelkern ready for review

File: apps/searchmanager/signalhandlers.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        self.itemid = itemid
+        self.itemtype = itemtype
+        super(IndexDocumentThread, self).__init__(**kwargs)
+
+    def run(self):
+        # Set sleep time to allow database unlocking and the commit of the save transaction
+        time.sleep(5)
+        # set the Search - Update Index Item API url for the current item.
+        api_url = settings.PC_SERVICES['references']['base_url'] + \
+            settings.PC_SERVICES['references']['updateindexitem'] + '/metric/' + str(self.itemid)
```
Comment (FabiApfelkern) [inline apps/searchmanager/signalhandlers.py]: @slomo Isn't that very specific to metric here?

**Comment by FabiApfelkern** [comment]
This may should be addressed for all Django apps. See: https://github.com/policycompass/policycompass/issues/142

**Comment by cbotsikas** [comment]
I'll have a look on this. was planing to do that anyway...

**Comment by cbotsikas** [comment]
deprecated due to https://github.com/policycompass/policycompass-services/pull/115

**Pull request closed without merging**